### PR TITLE
Hat accent layout and glyph tweaks.

### PR DESCRIPTION
**Pull request by jkdufair**
Well rider decided to clean up a lot of excess trailing whitespace I guess. See line 523 of `css/mathlive.less` and line 834 of `src/core/mathAtom.js` for the actual changes.

### CODE REVIEW

File: css/mathlive.less [hunk 1]
```diff
@@ -519,6 +519,23 @@
...
+            visibility: hidden;
+            position: relative;
+        }
+
+        .accent-body.accent-hat > span::after {
+            visibility: visible;
+            position: absolute;
+            top: 0.5em;
+            left: 0;
+            content: "ˆ";
```
Comment (builtbywill) [inline css/mathlive.less]: @jkdufair could we just replace the \u005e with \u02c6 in src/core/definitions? or do we also need the css changes in here?

Comment (jkdufair) [reply css/mathlive.less]: @builtbywill That was what I did originally, but it's pretty baked in in several places around the codebase and I didn't have any success. That one change didn't seem to do it. I can give it another go if you think this is too hacky. The newest version of mathlive handles it completely differently if we ever go there. And it's quite lovely.

Comment (builtbywill) [reply css/mathlive.less]: @jkdufair i just tried it out locally and it seemed to work just fine without the css changes

Comment (jkdufair) [reply css/mathlive.less]: @builtbywill Ok lemme give it another run then. Thanks for trying it out.

Comment (jkdufair) [reply css/mathlive.less]: @builtbywill Ok I replaced the character. I still added some css as it didn't look amazing.

Comment (builtbywill) [reply css/mathlive.less]: @jkdufair nice! seems good to me

**Review by builtbywill — APPROVED** [review]

File: css/mathlive.less (hunk 1 above)
Comment (builtbywill) [reply css/mathlive.less]: @jkdufair actually it looks like the font-size in your new css is messing with the layout

items with the hat are weirdly higher up off the baseline than normal

image

Comment (jkdufair) [reply css/mathlive.less]: @builtbywill Well that is just lovely. I took the font size (and top) out. I think it's good enough now. No more or less identifiable than any of the other accents. I think the remaining CSS is worth keeping.

Comment (builtbywill) [reply css/mathlive.less]: @jkdufair yeah those were the two that seemed to be unneeded / causing the problem

re-approved

**Review by builtbywill — APPROVED** [review]

**Review by tragessere — APPROVED** [review]

**Pull request merged**